CHANGELOG.md

```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Bug fixes
+  - Fixed a bug in type inference: #1022.
+
+### New library functions
+
+- New `ddlog_std` library functions:
+
+    ```
+    /* Convert any DDlog type into a string in a programmer-facing,
+     * debugging context.  Implemented by calling the `Debug::fmt()`
+     * method of the underlying Rust type. */
+    extern function to_string_debug(x: 'T): string
```

Comment:
should this be the default to_string method?

Reply:
Problem with default `to_string()` implementation is that at the moment neither DDlog nor Rust support specialization, so we cannot easily have a blanket implementation that appliez to all types and then overload it for a particular type.